src/metabase/util/ssh.clj

```diff
@@ -1,32 +1,63 @@
 (ns metabase.util.ssh
-  (:require [clojure.string :as str]
-            [clojure.tools.logging :as log]
+  (:require [clojure.tools.logging :as log]
             [metabase.util :as u])
-  (:import com.jcraft.jsch.JSch))
+  (:import java.io.ByteArrayInputStream
+           [org.apache.sshd.client.future AuthFuture ConnectFuture]
+           org.apache.sshd.client.session.ClientSession
+           org.apache.sshd.client.session.forward.PortForwardingTracker
+           org.apache.sshd.client.SshClient
+           [org.apache.sshd.common.config.keys FilePasswordProvider FilePasswordProvider$ResourceDecodeResult]
+           org.apache.sshd.common.session.SessionHolder
+           org.apache.sshd.common.util.GenericUtils
+           org.apache.sshd.common.util.io.resource.AbstractIoResource
+           org.apache.sshd.common.util.net.SshdSocketAddress
+           org.apache.sshd.common.util.security.SecurityUtils
+           org.apache.sshd.server.forward.AcceptAllForwardingFilter))
 
 (def ^:private default-ssh-timeout 30000)
 
+(def ^:private client
+  (doto (SshClient/setUpDefaultClient)
+    (.start)
+    (.setForwardingFilter AcceptAllForwardingFilter/INSTANCE)))
+
 (defn start-ssh-tunnel
   "Opens a new ssh tunnel and returns the connection along with the dynamically
-   assigned tunnel entrance port. It's the callers responsibility to call .disconnect
+   assigned tunnel entrance port. It's the callers responsibility to call `close-tunnel`
```

Comment:
Another thing that's not worth blocking merge to go change but for next time https://guide.clojure.style/#docstring-indentation

That docstring was already indented the wrong amount before this PR but if you change them might as well fix the indentation while you're at it